src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -37,12 +38,19 @@
 public class ThemeLoader {
 
     private static final String DEFAULT_PATH_MAIN_CSS = JabRefFrame.class.getResource("Base.css").toExternalForm();
-    private static final String CSS_SYSTEM_PROPERTY = System.getProperty("jabref.theme.css");
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeLoader.class);
+    private final String CSS_SYSTEM_PROPERTY;
     private final FileUpdateMonitor fileUpdateMonitor;
 
     public ThemeLoader(FileUpdateMonitor fileUpdateMonitor) {
         this.fileUpdateMonitor = Objects.requireNonNull(fileUpdateMonitor);
+
+        String cssFileName = JabRefPreferences.getInstance().get(JabRefPreferences.FX_THEME);
+        if (cssFileName != null) {
+            String themeName = JabRefFrame.class.getResource(cssFileName).getPath();
```

Comment:
Please use the nio methods for accessing the resource. Have a look at the exporter tests to see how